code/modules/clothing/suits/cardborg_suit.dm

```diff
@@ -0,0 +1,261 @@
+	icon_state = "cardborg_mining"
+	item_state = "cardborg_mining"
+	species_disguise = "High-tech mining robot"
+
+/obj/item/clothing/suit/cardborg/service
+	name = "green cardborg suit"
+	desc = "A full-body suit made out of ordinary cardboard boxes with various holes cut into them. This one has been spray-painted green."
+	icon_state = "cardborg_service"
+	item_state = "cardborg_service"
+	species_disguise = "High-tech service robot"
+
+/obj/item/clothing/suit/cardborg/medical
+	name = "cardborg suit"
+	desc = "A full-body suit made out of ordinary cardboard boxes with various holes cut into them. This one has been spray-painted blue."
+	icon_state = "cardborg_medical"
+	item_state = "cardborg_medical"
+	species_disguise = "High-tech medical robot"
+
+/obj/item/clothing/suit/cardborg/janitor
+	name = "purple cardborg suit"
+	desc = "A full-body suit made out of ordinary cardboard boxes with various holes cut into them. This one has been spray-painted purple."
+	icon_state = "cardborg_janitor"
+	item_state = "cardborg_janitor"
+	species_disguise = "High-tech janitor robot"
+
+/obj/item/clothing/suit/cardborg/xeno
+	name = "black cardborg suit"
+	desc = "A full-body suit made out of ordinary cardboard boxes with various holes cut into them. This one has been spray-painted white."
+	icon_state = "cardborg_xeno"
+	item_state = "cardborg_xeno"
+	species_disguise = "High-tech alien-hunting robot"
+
+/obj/item/clothing/suit/cardborg/deathbot
+	name = "black cardborg suit"
+	desc = "A full-body suit made out of ordinary cardboard boxes with various holes cut into them. This one has been spray-painted black."
+	icon_state = "cardborg_deathbot"
+	item_state = "cardborg_deathbot"
+	species_disguise = "High-tech killer robot"
+
+/*
+ / Disguise code.
+*/
+/obj/item/clothing/head/cardborg/equipped(mob/living/user, slot)	// Attempt to disguise when you put on the helmet.
+	..()
+	if(ishuman(user) && slot == SLOT_HUD_HEAD)
+		var/mob/living/carbon/human/H = user
+		if(istype(H.wear_suit, /obj/item/clothing/suit/cardborg))
+			var/obj/item/clothing/suit/cardborg/CB = H.wear_suit
+			CB.apply_borg_disguise(user, src)
+
+/obj/item/clothing/head/cardborg/dropped(mob/living/user)			// You stop being a robot if you remove your helmet.
+	..()
+	user.remove_alt_appearance("borg_disguise_variant")
+
+/obj/item/clothing/suit/cardborg/equipped(mob/living/user, slot)	// Attempt to disguise when you put on the suit.
+	..()
+	if(slot == SLOT_HUD_OUTER_SUIT)
+		apply_borg_disguise(user)
+
+/obj/item/clothing/suit/cardborg/dropped(mob/living/user)			// You stop being a robot if you remove your suit.
+	..()
+	user.remove_alt_appearance("borg_disguise_variant")
+
+/obj/item/clothing/suit/cardborg/proc/apply_borg_disguise(mob/living/carbon/human/H, obj/item/clothing/head/cardborg/borghead)
+	if(!istype(H))
+		return
+	if(!borghead)
+		borghead = H.head									// This actually stops the disguise from applying just from having it in your hands AGHHHH!
```

Comment:
This comment doesn't seem correct? You are checking if `equipped` is applying it to the correct HUD slot so picking it up should be fine?